Client/create/src/Pin.jsx

```diff
@@ -32,6 +33,10 @@ module.exports = React.createClass({
       this.setState({editLocationMode: true});
     }
   },
+  removePin:function(){
+    Actions.removePin(this.props.pinIndex);
```

Comment:
This currently does not update the distance and time estimates. Add something like
`
var huntInfo = {
        huntTimeEst: gMap.getDuration(),
        huntDistance: gMap.getDistance()
      };
 Actions.updateHuntAtKey(huntInfo, 'huntInfo');
`
below line 37 and it should address this.